Habitica/src/main/java/com/habitrpg/android/habitica/ui/menu/MainDrawerBuilder.java

```diff
@@ -32,6 +33,7 @@
     // Change the identificationIDs to the position IDs so that its easier to set the selected entry
     public static final int SIDEBAR_TASKS = 0;
     public static final int SIDEBAR_SKILLS = 1;
+    public static final int SIDEBAR_INBOX = 15;
```

Comment:
can you change the IDs to be corresponding to their position? (so inbox is 3, tavern 4, etc.)